### ISSUE TITLE
[BITV/A11Y] No aria-expandCannot control

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/files_pdfviewer#636]

**Comment by danxuliu** [comment]
[`aria-expanded` for the sidebar, search and secondary toolbar was fixed in PDF.js 2.8.335](https://github.com/mozilla/pdf.js/commit/374da648dd5df2fa7616d06d4f46f97bdd892c13), so the original report should be fixed since [Nextcloud 22.2.0](https://github.com/nextcloud/files_pdfviewer/blob/v22.2.0/package.json#L30). However, the issue was added again with the annotation buttons in a later version.

[`aria-expanded` for highlight, free text, ink and stamp annotation buttons was fixed in PDF.js 4.7.76](https://github.com/mozilla/pdf.js/commit/6f40ed729bf8b504bb47b1e28c4c26fb4601435f), so an update to that or a later version should fix it again.
